test/integration/targets/nxos_nxapi_transports/tests/nxapi/assert_transports.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+- nxos_command:
+    commands:
+      - show hostname
+  connection: httpapi
+  vars:
+    ansible_httpapi_use_ssl: yes
+    ansible_httpapi_port: 443
```

Comment:
@tstoner @mikewiebe I am wondering if the port is even needed here. 
since `use_ssl` is True, https port by default is 443 code https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/connection/httpapi.py#L202-L204
doc https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/connection/httpapi.py#L29

Reply:
Agreed.  We will let the default behavior determine the port.  Will remove and push up changes.